### PR TITLE
sinkv2(ticdc): add some comments

### LINKED ISSUE #7437: TiCDC: refine log output and comments.
### Is your feature request related to a problem?

To be do.

### Describe the feature you'd like

as title.

### Describe alternatives you've considered

_No response_

### Teachability, Documentation, Adoption, Migration Strategy

_No response_

### PR DESCRIPTION
**Pull request by asddongmen**
### What problem does this PR solve?

Issue Number: close #7437 

#### Tests 

 - No code

### Release note 

```release-note
None
```

### CODE REVIEW

File: cdc/processor/pipeline/sink.go [hunk 1]
```diff
@@ -272,23 +272,19 @@ func shouldSplitUpdateEvent(updateEvent *model.PolymorphicEvent) bool {
...
 		col := updateEvent.Row.Columns[i]
 		preCol := updateEvent.Row.PreColumns[i]
 		if col != nil && col.Flag.IsHandleKey() && preCol != nil && preCol.Flag.IsHandleKey() {
-			handleKeyCount++
 			colValueString := model.ColumnValueString(col.Value)
 			preColValueString := model.ColumnValueString(preCol.Value)
-			if colValueString == preColValueString {
-				equivalentHandleKeyCount++
+			// If one handle key columns is updated, we need to split the event row.
+			if colValueString != preColValueString {
```
Comment (asddongmen) [inline cdc/processor/pipeline/sink.go:282]: /cc @hi-rustin  Please help me check if this change is ok?

**Review by hi-rustin — APPROVED** [review]
Please refine your PR description. Thanks!

File: cdc/sinkv2/eventsink/txn/worker.go [hunk 2]
```diff
@@ -125,7 +125,7 @@ func (w *worker) runBackgroundLoop() {
 			zap.String("changefeedID", w.changefeed),
 			zap.Int("workerID", w.ID))
 
-		timer := time.NewTicker(w.flushInterval)
+		ticker := time.NewTicker(w.flushInterval)
```
Comment (amyangfei) [inline cdc/sinkv2/eventsink/txn/worker.go:128]: should call `ticker.Stop` in defer.

Comment (asddongmen) [reply cdc/sinkv2/eventsink/txn/worker.go:128]: got it. thanks.

**Review by amyangfei — APPROVED** [review]

**Pull request merged**